cli-commands/forgae-node/node.js

```diff
@@ -89,7 +91,28 @@ async function fundWallet(client, recipient) {
 
 }
 
+function hasNodeConfigFiles() {
+  const neededConfigFile = nodeConfig.dockerConfiguration.configFileName;
+  const configFilePath = path.resolve(process.cwd(), neededConfigFile);
+  let isDockerFolderExist = fs.existsSync(configFilePath);
```

Comment:
isDockerFolderExist is incorrect gramatical name lets rename it to `dockerFolderExists`